controllers/controllers.js

```diff
@@ -81,4 +81,18 @@ exports.patchArticle = (req, res, next) =>{
         res.status(200).send(newArticle)
     }).catch((err)=>{next(err)})
 
+}
+
+exports.removeComment = (req, res, next) =>{
+    const {comment_id} = req.params;
+
+   console.log(typeof comment_id, "<<")
+    const commentExists = checkCommentExists(comment_id)
```

Comment:
You don't need this, if you use returning * in your query you'll see what rows were affected, if that count is 0 then that comment didn't exist and you can reject.